src/ResourceParser.h

```diff
@@ -133,6 +133,13 @@ namespace snowcrash {
                             attributes.sourceMap.name.sourceMap = out.sourceMap.name.sourceMap;
                         }
                     }
+                    else {
+
+                        mdp::CharactersRangeSet sourceMap = mdp::BytesRangeSetToCharactersRangeSet(node->sourceMap, pd.sourceData);
+                        out.report.error = Error("attributes can not be added to a resource which doesn't have a name",
```

Comment:
Furthermore this is in contradiction with API Blueprint specification – https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md#def-resource-section : 

> Attributes defined in the scope of a Resource section represent Resource attributes. If the resource is defined with a name these attributes may be referenced in Attributes sections.